test/e2e/service.go

```diff
@@ -397,7 +397,7 @@ var _ = framework.KubeDescribe("Services", func() {
 		hostExec := framework.LaunchHostExecPod(f.Client, f.Namespace.Name, "hostexec")
 		// Even if the node-ip:node-port check above passed, this hostexec pod
 		// might fall on a node with a laggy kube-proxy.
-		cmd := fmt.Sprintf(`for i in $(seq 1 300); do if ss -ant46 'sport = :%d' | grep ^LISTEN; then exit 0; fi; sleep 1; done; exit 1`, nodePort)
+		cmd := fmt.Sprintf(`for i in $(seq 1 300); do if ss -ant46 'sport = :%d' | grep ^LISTEN; then exit 0; fi; sleep 1; done; ss -ant46 'sport = :%d'; exit 1`, nodePort, nodePort)
```

Comment:
so i already dislike how we pile stuff into a bash onliner then run around trying to add clarity, but now we're piling debug stuff onto it? :)

Would be nice to clean it up sometime. I don't have ideas of a better short term fix, though.